frontends/web/src/routes/account/walletconnect/utils.ts

```diff
@@ -118,4 +118,16 @@ export async function pair(params: { uri: string }) {
     throw new Error('Please use a new URI!');
   }
   await web3wallet?.core.pairing.pair({ uri });
-}
\ No newline at end of file
+}
+
+export const getAddressFromEIPString = (address: string) => {
+  const parts = address.split(':');
+  return parts.length > 2 ? parts[2] : '';
```

Comment:
do those EIP strings always have a colon `:`?

Reply:
In this case, yes I assume so. Assumption comes from here (we "write" it this way):

https://github.com/digitalbitbox/bitbox-wallet-app/blob/87c1987c8247aee791e5a1f731e762ac206aed3e/frontends/web/src/routes/account/walletconnect/components/incoming-pairing/incoming-pairing.tsx#L61 , or in Tom's PoC:

https://github.com/digitalbitbox/bitbox-wallet-app/blob/87c1987c8247aee791e5a1f731e762ac206aed3e/frontends/web/src/routes/account/walletconnect/index.tsx#L269
(chain is for example `eip155:1`)


Just to confirm, @Tomasvrba, could you verify that the address is indeed always formatted the same; which is after the second colon (e.g: `eip155:1:walletAddress`), please?  thanks. 🙏 

_note: can also use regex instead if we prefer that._

Reply:
That's correct, see: https://docs.walletconnect.com/2.0/specs/clients/sign/namespaces#25-session-namespaces-may-contain-multiple-accounts-for-one-chain

Reply:
> can also use regex instead if we prefer that.

I think it is fine, no need for regex.